File: 2019/dayTemplate/dayX.go
package main

import (
	"fmt"
	"github.com/adventofcode/2019/common"
)

func main() {
	inputAsStr := string(common.ReadBytes("./dayX/input.txt"))

	result1 := RunProgram(inputAsStr)
	result2 := RunProgram(inputAsStr)

	fmt.Printf("Part 1 is: %d \n", result1)
	fmt.Printf("Part 2 is: %d \n", result2)
}

func RunProgram(input string) int {
	return 0
}

File: 2022/src/main/kotlin/DayX.kt
object DayX {

  @JvmStatic
  fun main(args: Array<String>) {
    println("Starting Day X - part 1")
    val input = this::class.java.getResource("/input/dayX.txt")?.readText() ?: ""
    val result1 = part1(input)
    println("Result for DayX - part 1: $result1")

    val result2 = part2(input)
    println("Result for DayX - part 2: $result2")
  }

  fun part1(input: String): Int {
    TODO()
  }

  fun part2(input: String): Int {
    TODO()
  }
}
